apiv2/views.py

```diff
@@ -1229,7 +1228,7 @@ def get_description(cls):
                % (prepend_base('/docs/api'), 'index.html')
 
     def get(self, request,  *args, **kwargs):
-        logger.info(self.log_message('api_root'))
+        api_logger.info(self.log_message('api_root'))
```

Comment:
out of interest, the log message format constructed by this is different to the format constructed with json and `()`. Is this something that should be unified eventually?

Reply:
Yes, it is different because initially we were only sending "structured" data from the API logs, then I added a "generic" way of doing it. It could be unified at some point.

Reply:
cool, let's open a ticket for it then. It's a good continuation of my comment about improving this in the future.